chef/automate
# Local User events

### :nut_and_bolt: Description: What code changed, and why?

Adding create, update, and delete events for Automate local users.

### :chains: Related Resources

https://github.com/chef/automate/pull/4108

### :+1: Definition of Done
Events are created when creating, deleting, and updating Automate's local users. 

### :athletic_shoe: How to Build and Test the Change
1. `build components/automate-gateway && start_all_services`
1. Start the local UI or `rebuild componets/automate-ui`
1. Go to https://a2-dev.test/settings/users and create a new user. 
1. Update that users display name
1. Delete the user. 
1. Go to https://a2-dev.test/dashboards/event-feed and ensure there are three events for each action. 

### :white_check_mark: Checklist


### :camera: Screenshots, if applicable
<img width="1554" alt="Screen Shot 2020-08-20 at 1 19 10 PM" src="https://user-images.githubusercontent.com/1679247/90821567-f8571200-e2e7-11ea-852d-be2145d2174d.png">
